<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HWarehouseDetails</class>
 <widget class="QWidget" name="HWarehouseDetails">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modifica Operazione</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/Resources/Monitoring64.png</normaloff>:/Resources/Monitoring64.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ID Operazione:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leID">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 127);</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Data:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="deData">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="currentSection">
          <enum>QDateTimeEdit::DaySection</enum>
         </property>
         <property name="displayFormat">
          <string>dd/MM/yyyy hh:mm:ss</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Operatore:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leOperatore">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 127);</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="lbidlotto">
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Prodotto:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leProdotto">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 127);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Lotto:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leLotto">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 127);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Giacenza:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leGiacenza">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 127);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Scadenza:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="deScadenza">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 127);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="currentSection">
        <enum>QDateTimeEdit::MonthSection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Lotto Fornitore</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leLotFornitore">
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Lotto Esterno</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leEAN">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 127);</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Azione:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbAzione"/>
       </item>
       <item>
        <widget class="QLineEdit" name="leQuantita"/>
       </item>
       <item>
        <widget class="QComboBox" name="cbUM"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>500</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Note:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="ptNote">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="pbSave">
       <property name="text">
        <string>Salva</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/Resources/save-icon.png</normaloff>:/Resources/save-icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbClose">
       <property name="text">
        <string>Chiudi</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/Resources/Actions-window-close-icon.png</normaloff>:/Resources/Actions-window-close-icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
